src/main/resources/db/migration/V1__Create_course_table.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS courses (
+    id VARCHAR NOT NULL PRIMARY KEY,
+    institute VARCHAR NOT NULL,
+    name VARCHAR NOT NULL,
+    timezone VARCHAR NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP
+);
+
+-- TODO: make account_id foreign key to the accounts table
+-- TODO: make email and course_id a composite primary key
+CREATE TABLE IF NOT EXISTS instructors (
+    id VARCHAR NOT NULL PRIMARY KEY,
+    account_id VARCHAR,
+    course_id VARCHAR UNIQUE,
+    email VARCHAR NOT NULL,
+    name VARCHAR NOT NULL,
+    registration_key VARCHAR NOT NULL,
+    displayed_name VARCHAR NOT NULL,
+    role VARCHAR NOT NULL,
+    instructor_privileges TEXT NOT NULL,
```

Comment:
Since `VARCHAR` and `TEXT` are essentially the same, wanted to ask reasons for using `TEXT` here?

Reply:
> is it possible to validate data definition in hibernate? wondering if this is done automatically.

I believe it can be configured to be automatic by setting the `hibernate.hbm2ddl.auto` option to `validate`, will need to experiment to find out.

Reply:
> Since `VARCHAR` and `TEXT` are essentially the same, wanted to ask reasons for using `TEXT` here? And also in the data model design document, this is of type `JSON`, not sure are we going to following that design?

Yes I think we should look into the benefits of using `json` or `jsonb` as the data type for `instructor_privileges`, as this PR mainly concerns database migrations I didn't delve into that.

Reply:
> Since `VARCHAR` and `TEXT` are essentially the same, wanted to ask reasons for using `TEXT` here?

They are the same with the exception that `VARCHAR` allows setting of maximum length, with default length 255. Since the length of instructor privileges frequently goes above the default value, `TEXT` would be easier to use here.

I agree with looking into using JSON, maybe we can make the modifications in a separate PR down the line.